Journal/Journal/CoreDataStack.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CoreDataStack.swift
+//  Journal
+//
+//  Created by Mitchell Budge on 6/3/19.
+//  Copyright © 2019 Mitchell Budge. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+class CoreDataStack {
+    
+    static let shared = CoreDataStack()
+    
+    lazy var container: NSPersistentContainer =  {
+        let container = NSPersistentContainer(name: "Journal")
```

Comment:
Here is your problem, you need to call it the name of your CoreData model. In your case "Entry"